src/main/java/delete_event_use_case/DeleteEventOutputData.java

```diff
@@ -0,0 +1,29 @@
+package delete_event_use_case;
+
+public class DeleteEventOutputData {
+    private String label;
+    private int dayIndex;
+    private boolean success = true;
+    private String errorMessage = "";
```

Comment:
I decided that the errorMessage should be set to null, so this should be changed accordingly. Thanks!

Reply:
I'll change that. Thanks!